catalog/ui/catalog-ui-search/src/main/webapp/component/location-old/location-old.view.js

```diff
@@ -426,10 +433,77 @@ define([
             this.$('#utmZone').multiselect(singleselectOptions);
             this.$('#utmHemisphere').multiselect(singleselectOptions);
 
+            var keywordProperty = new Property({
```

Comment:
It'd be worth putting this block into it's own method `showKeywordPropertyView` so the logic is isolated, and so we can reshow the keyword property on it's own if need be.  Not critical or anything though.